scripts/bert/dataset.py

```diff
@@ -169,19 +175,19 @@ def __call__(self, line):
         # the entire model is fine-tuned.
         tokens = []
         segment_ids = []
-        tokens.append('[CLS]')
+        tokens.append(BERTVocab.CLS_TOKEN)
```

Comment:
We should pass a bert_vocab object as input to this class and retrieve vocab.cls_token. Same comment for other lines